earth_enterprise/src/installer/install_fusion.sh

```diff
@@ -250,7 +250,7 @@ check_prereq_software()
 		check_prereq_software_retval=1
 	fi
 
-	if ! software_check "$script_name" "python2.7" "python-2.7.*" "python27-*"; then
```

Comment:
I think instead of separating Cent OS from RHEL, and using it as a separation between Cent OS / RHEL 7 and Cent OS / RHEL 6 we should accept both Python 2.6 and Python 2.7 here, since GEE should now run with both.

We just need to change the `grep` expression to match both '2.6' and '2.7'.